packages/contracts/hardhat.config.ts

```diff
@@ -58,24 +58,33 @@ const config: HardhatUserConfig = {
   },
   solidity: {
     compilers: [
-      {
-        version: '0.7.6',
-      },
+      // Compilers must be listed in ascending order, or things break
```

Comment:
Wait, really?

Reply:
yep!

Reply:
Or... maybe not. 
Might have been something else breaking....